srcpkgs/scenefx/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'scenefx'
+pkgname=scenefx
+version=0.1
+revision=1
+archs="i686 x86_64"
+build_style=meson
+configure_args="-Dwerror=false -Db_ndebug=false"
+hostmakedepends="pkg-config wayland-devel scdoc"
+makedepends="wlroots0.17-devel"
+depends="libdrm pixman"
+short_desc="Drop-in replacement for the wlroots scene API"
+maintainer="Christopher K. "Shmish" Schmitt <me@shmish.dev>"
```

Comment:
needs to be single-quotes for the nickname

Reply:
Noted and corrected.